executor/executor.go

```diff
@@ -909,6 +918,55 @@ func (e *SelectLockExec) Open(ctx context.Context) error {
 	return nil
 }
 
+func (e *SelectLockExec) generatePartitionedTableColumnMap(pt table.PartitionedTable, is infoschema.InfoSchema) error {
+	// Get Table Name and DB name
+	tblInfo := pt.Meta()
+	dbInfo, ok := is.SchemaByTable(tblInfo)
+	if !ok {
+		return errors.Trace(errors.Errorf("Cannot get schema info for table %s", tblInfo.Name.O))
+	}
+	colNamePrefix := fmt.Sprintf("%s.%s.", dbInfo.Name.L, tblInfo.Name.L)
+	cols := pt.Cols()
```

Comment:
Do we need to use `VisibleCols` instead of `Cols`? @bb7133

Reply:
Yes, I think so.